content/envs/_index.en.md

```diff
@@ -105,35 +105,7 @@ DIAMBRA Arena Environments usage follows the standard RL interaction framework:
 
 The shortest snippet for a complete basic execution of an environment consists of just a few lines of code, and is presented in the code block below:
 
-```python {linenos=inline}
- # DIAMBRA Arena module import
- import diambra.arena
-
- # Environment creation
- env = diambra.arena.make("doapp")
-
- # Environment reset
- observation = env.reset()
-
- # Agent-Environment interaction loop
- while True:
-     # (Optional) Environment rendering
-     env.render()
-
-     # Action random sampling
-     actions = env.action_space.sample()
-
-     # Environment stepping
-     observation, reward, done, info = env.step(actions)
-
-     # Episode end (Done condition) check
-     if done:
-         observation = env.reset()
-         break
-
- # Environment close
- env.close()
-```
+{{< github_code "https://raw.githubusercontent.com/diambra/arena/main/examples/diambra_arena_gist.py" >}}
```

Comment:
@discordianfish this is an example on how all hand coded snippets have been removed from the docs source code, and automatically obtained from the repos.